Src/Plugins/Security/SEC101.SecurePlaintextSecrets.json

```diff
@@ -1,318 +1,319 @@
 {
   "ValidatorsAssemblyName": "Security.dll",
   "SharedStringsFileName": "Security.SharedStrings.txt",
+  "ExtensionName": "DoNotExposePlaintextSecrets",
+  "Guid": "6e04e237-f14b-4ff9-9cd8-e037a10cb510",
   "Definitions": [
     {
       "Id": "SEC101",
-      "Name": "DoNotExposePlaintextSecrets",
       "Level": "Warning",
       "FileNameDenyRegex": "$BinaryFiles",
       "Description": "Do not expose plaintext (or base64-encoded plaintext) secrets in versioned engineering content.",
       "Message": "'{0:truncatedSecret}' is {1:validationPrefix}{2:encoding}{3:secretKind}{4:validationSuffix}{5:validatorMessage}.",
       "MatchExpressions": [
         {
           "Id": "SEC101/002",
-          "Name": "DoNotExposePlaintextSecrets/GoogleOAuthCredentials",
+          "Name": "GoogleOAuthCredentials",
```

Comment:
>GoogleOAuthCredentials [](http://example.com/codeflow?start=19&length=22)

Hooray! Now that we have an extension name, we don't have to embed it in the rule id!

But heads up, everyone who is still using the rule name as a key: all of these names are changing.

Consumers who properly use the stable opaque id won't have an issue. ;)